documentation/docs/steps/transportRequestUploadCTS.md

```diff
@@ -138,6 +138,18 @@ transportRequestReqIDFromGit( script: this )
 transportRequestUploadCTS( script: this, ... )
 ```
 
+## General purpose pipeline release stage
+
+The step can also be configured via General purpose pipeline in Release stage using config.yml. The configuration will look as below:
+
+```groovy
+stages:
+  Release:
+    transportRequestUploadCTS: true
+```
+
+This will initialize the step within the Release stage of pipeline and will upload the desired application (SAPUI5/OPENUI5) to the SAPUI5 ABAP repository.
```

Comment:
```suggestion
This will initialize the step within the Release stage of the pipeline and will upload the desired application (SAPUI5/OPENUI5) to the SAPUI5 ABAP repository.
```